fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -288,7 +289,10 @@ rocksdb::ColumnFamilyOptions getCFOptions() {
 	}
 
 	if (SERVER_KNOBS->ROCKSDB_BLOCK_CACHE_SIZE > 0) {
-		bbOpts.block_cache = rocksdb::NewLRUCache(SERVER_KNOBS->ROCKSDB_BLOCK_CACHE_SIZE);
+		if (rocksdb_block_cache == nullptr) {
```

Comment:
Just curios, when is this getting recreated?
I see this method and cache creation is getting called only in OpenAction, CloseAction, RestoreAction which are very rare events right?

Reply:
Yes. Seems this should be a tiny fix.

Reply:
Is it possible to store the Options object somewhere so could reuse it?